src/main/java/uk/gov/hmcts/ccd/domain/service/createevent/MidEventCallback.java

```diff
@@ -90,6 +90,19 @@ public JsonNode invoke(String caseTypeId,
         return dataJsonNode(content.getData());
     }
 
+    private void removeNextPageFieldData(CaseDetails currentCaseDetails, Integer order, String caseTypeId, String eventId) {
+        if (order != null) {
```

Comment:
in theory page display order is optional. Even thought we recommend to always set it. The ordering is guaranteed only if a display order is defined. Otherwise I think pages are ordered by page id or in the order they are defined on the spreadsheet (can't recall exactly). In this case we won't be removing the next pages right?

Reply:
I have verified all the prod files, PageDisplayOrder is defined for most of the definitions except for some fields in below definition files.
**CCD_Publiclaw_BulkScanningExceptionRecord_V00.01-PROD
ccd-fpl-prod**